components/prize_pool/commons/prize_pool_base.lua

```diff
@@ -484,10 +484,25 @@ function BasePrizePool._comparePrizes(x, y)
 	return sortX == sortY and x.index < y.index or sortX < sortY
 end
 
+function BasePrizePool:_shouldDisplayPrizeSummary()
+	-- if prizeSummary is disabled do not show it
+	if not self.options.prizeSummary then
+		return false
+	end
+
+	local baseMoney = tonumber(Variables.varDefault('tournament_prizepool_' .. BASE_CURRENCY:lower())) or 0
+	-- if we have currency conversion or entered usd values display it
+	-- if we have baseMoney and it being not 0 display it
+	-- should we add a check against tournament_parent wiki var to display it if we have section preview???
```

Comment:
.

Reply:
the `tournament_parent` is not available in section preview since it gets set by infobox/hdb
so if it is not present we could just display the summary again
(mind that if we decide to display it if `tournament_prizepoolusd` var is empty then this is redundant anyways)